tagger/safari_12_1_stp.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"strings"
+	"time"
+
+	"cloud.google.com/go/datastore"
+	mapset "github.com/deckarep/golang-set"
+
+	"github.com/web-platform-tests/data-migration/processor"
+	"github.com/web-platform-tests/wpt.fyi/shared"
+)
+
+// safari12STPLabeller ensures that Safari 12.1 runs are labelled
+// 'experimental' (and not 'stable') if they ran before Safari 12.1 was
+// actually released to stable.
+type safari12STPLabeller struct{}
+
+func (e safari12STPLabeller) ShouldProcessRun(run *shared.TestRun) bool {
+	if run.BrowserName != "safari" {
+		return false
+	}
+	if !strings.HasPrefix(run.BrowserVersion, "12.1") {
+		return false
+	}
+	// We ran Safari 12.1 as STP from 2018-08-07 until 2019-01-05. It then
+	// released as stable 2019-04-09. So anything after February (to pick
```

Comment:
What is the earliest run after this cutoff point?

Reply:
Sorry, a bit of misleading wording.

Safari 12.1 was released on 2019/03/25 (https://en.wikipedia.org/wiki/Safari_version_history#Safari_12). The first stable WPT run (aka the earliest run) was 2019/04/09.